frontend/src/views/CatalogView.tsx

```diff
@@ -0,0 +1,110 @@
+    return (
+        <div className="flex flex-col items-center gap-4 w-full h-full">
+            <h1 className="text-3xl font-semibold text-gray-700">View EDC Catalog</h1>
+            <div className="flex items-end gap-5">
+                <Input
+                    label="EDC URL"
+                    type="text"
+                    id="edc-url"
+                    placeholder="Enter URL"
+                    margin="none"
+                    onChange={(event) => (urlRef.current = event.target.value)}
+                />
+                <div className="mb-3">
+                    <LoadingButton
+                        label="Get Catalog"
+                        loadIndicator="Loading..."
+                        loading={isLoadingCatalog}
+                        onButtonClick={() => setEdcUrl(urlRef?.current)}
+                    />
+                </div>
+            </div>
+            {catalog ? (
+                <ul className="flex flex-col gap-5 w-[64rem]">
+                    {catalog.map((item, index) => (
+                        <Card className="p-5">
+                            <li key={index}>
+                                <h2 className="text-xl font-semibold">Catalog Item</h2>
+                                <div className="flex w-full justify-start gap-4">
+                                    <div className="grid grid-cols-2">
+                                        <h3 className="font-semibold">Asset ID: </h3>"{item.assetId}"
+                                        <h4 className="font-semibold">Asset type: </h4>"{item.assetType}"
+                                        <h4 className="font-semibold">Asset action: </h4>
+                                        {item.permission['odrl:action']['odrl:type']} {item.permission['odrl:target']}
+                                        <h4 className="font-semibold">Asset condition: </h4>
+                                        {item.permission['odrl:constraint']['odrl:leftOperand'] + ' '}
+                                        {getCatalogOperator(item.permission['odrl:constraint']['odrl:operator']['@id']) + ' '}
+                                        {item.permission['odrl:constraint']['odrl:rightOperand']}
+                                    </div>
+                                    <div className="flex flex-col w-1/2">
+                                        <OperationList title="The following prohibitions are defined:" operations={item.prohibitions} />
+                                        <OperationList title="The following obligations are defined:" operations={item.obligations} />
+                                    </div>
+                                </div>
+                            </li>
+                        </Card>
+                    ))}
+                </ul>
+            ) : null}
+            {catalogError != null ? <div className="text-red-500 py-5">There was an error retrieving the Catalog from {edcUrl}</div> : null}
```

Comment:
If error came up because of not retrieiving information and you then enter a correct url, the error is still shown after successful catalog request, too